frontend/front/src/pages/Public/Components/ThanksForSubmission.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Alert, Button } from "@mui/material";
+import { useNavigate } from "react-router-dom";
+
+export const ThanksForSubmission = () => {
+  const navigate = useNavigate();
+  return (
+    <Alert
+      severity="success"
+      action={
+        <Button
+          color="inherit"
+          size="small"
+          onClick={() => {
+            navigate("/public");
```

Comment:
Not 100% sure how navigating to `/public` for example is going to interact with the deployed website